test/unit_test/api/db/test_gaussdb_peewee.py

```
@@ -0,0 +1,421 @@
+        monkeypatch.setenv("GAUSSDB_METADATA_PORT", "19995")
+        monkeypatch.setenv("GAUSSDB_METADATA_USER", "zws")
+        monkeypatch.setenv("GAUSSDB_METADATA_PASSWORD", "secret")
+        monkeypatch.setenv("GAUSSDB_METADATA_DBNAME", "zws_test")
+        monkeypatch.setenv("GAUSSDB_METADATA_SCHEMA", "ragflow_meta")
+
+        config = load_database_config("GaussDB")
+
+        assert config["host"] == "127.0.0.1"
+        assert config["port"] == 19995
+        assert config["user"] == "zws"
+        assert config["password"] == "secret"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "password"

(S105)